omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/core/property/EnumProperty.java

```diff
@@ -10,11 +10,22 @@
 package org.openmrs.module.webservices.docs.swagger.core.property;
 
 import io.swagger.models.properties.StringProperty;
-import org.openmrs.module.webservices.docs.swagger.SwaggerSpecificationCreator;
+
+import java.util.Arrays;
+import java.util.List;
 
 public class EnumProperty extends StringProperty {
 	
 	public EnumProperty(Class<? extends Enum<?>> e) {
-		_enum(SwaggerSpecificationCreator.getEnumsAsList(e));
+		_enum(getEnumsAsList(e));
+	}
+
+	public static String[] getEnums(Class<? extends Enum<?>> e) {
+		return Arrays.toString(e.getEnumConstants())
+		        .replaceAll("^.|.$", "").split(", ");
+	}
+
+	public static List<String> getEnumsAsList(Class<? extends Enum<?>> e) {
```

Comment:
Once you replaced all instances of `EnumProperty.getEnumsAsList`, you can change the two methods:
from `public static` to `private`